### ISSUE TITLE
/_stats broken after closing index with alias

### ISSUE THREAD

**Issue by brainsiq** [elastic/elasticsearch#15744]
If an aliased index is closed then the _stats endpoint returns an error

```
curl http://localhost:9200/_stats

{"error":{"root_cause":[{"type":"index_closed_exception","reason":"closed","index":"test_index"}],"type":"index_closed_exception","reason":"closed","index":"test_index"},"status":403}
```

This does not happen if the alias is removed first. I think that this should not error, it should just exclude the closed index from the stats as normal?

**Comment by clintongormley** [comment]
Related to #15730 and #14982

**Comment by javanna** [comment]
Same as described in https://github.com/elastic/elasticsearch/issues/15730#issuecomment-171243705 .

The exception returned by `GET _stats` has been resolved by #15882, given that not specifying indices is considered a wildcard expressions. `GET _stats?expand_wildcards=open,closed` does return an exception as the `expand_wildcards` option is honoured.

If you do specify an alias that points to some closed index, you get back an exception by default: `GET alias/_stats` but `ignore_unavailable` allows to control whether closed indices should be ignored or not: `GET alias/_stats?ignore_unavailable=true` doesn't return exception.

I think we can close this one, the actual bug was same as #13278 which was fixed by #15882 .
